Trading/Mode/daily_trading_mode/daily_trading_mode.py

```diff
@@ -337,7 +337,8 @@ async def create_new_orders(self, symbol, final_note, state, **kwargs):
 
     async def internal_callback(self, trading_mode_name, cryptocurrency, symbol, time_frame, final_note, state, data):
         # creates a new order (or multiple split orders), always check self.can_create_order() first.
-        await self.create_order_if_possible(symbol, final_note, state)
+        if not await self.create_order_if_possible(symbol, final_note, state):
```

Comment:
It could be more pythonic with an exception here

Reply:
👍 done in https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/283